vinculacion/_linking_algorithm.py

```
@@ -0,0 +1,575 @@
+import pandas as pd
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)